### DISCUSSION TITLE
How to avoid validating not changed values

### DISCUSSION THREAD

**Discussion by techguydev** [livewire/livewire#1298] [4. General]
I am creating a simple profile form that has text fields and image field, I am able to update it already with new data.

Although I see that If I try to update just one text field the  image field keeps showing the validation error even if I did not made any change, the same thing happens with the text fields, for example the ones that require a unique value in a DB.

How is it supposed to be handled?, I think that there should be kind of  dinamic validation

I am mounting the data:

 public function mount()
    {       
        $this->slug = auth()->user()->slug;
        $this->name = auth()->user()->name;
        $this->phone = auth()->user()->phone;
        $this->bio = auth()->user()->bio;
    }

and I am validating like this in the save method:

 $this->validate([
             'newphoto' => 'image|max:2024',
             'slug' => 'required|unique:users|max:20',
             'name' => 'required',
             'phone' => 'required',
             'bio' => 'max:300',
        ]);

**Comment by nuernbergerA** [comment]
https://laravel-livewire.com/docs/input-validation#real-time-validation

```php
public function updated($field)
{
    $this->validateOnly($field, [
        'name' => 'min:6',
        'email' => 'email',
    ]);
}
```
